src/modules/navigator/rtl_params.c

```diff
@@ -110,10 +110,10 @@ PARAM_DEFINE_FLOAT(RTL_MIN_DIST, 5.0f);
  * Fly straight to the return location or planned mission landing and land there or
  * use the planned mission to get to those points.
  *
- * @value 0 Return home via direct path
- * @value 1 Return to a planned mission landing, if available, via direct path, else return to home via direct path
- * @value 2 Return to a planned mission landing, if available, using the mission path, else return to home via the reverse mission path
- * @value 3 Return via direct path to closest destination: home, mission landing pattern or safe point
+ * @value 0 Return to closest safe point (home or rally point) via direct path.
+ * @value 1 Return to closest safe point (planned mission landing or rally point), if available, else return to home.
+ * @value 2 Same as above (2nd option), but use the mission path (for return to mission landing), the reverse mission path (home) and direct path (rally point).
```

Comment:
Same question as above for home vs rally points. Will affect how this is worded so will wait on your response.